src/plugins/workspace/public/components/workspace_list/index.tsx

```diff
@@ -65,129 +131,198 @@ export const WorkspaceList = ({ registeredUseCases$ }: WorkspaceListProps) => {
+        >
+          {/* Here I need to set width mannuly as the tooltip will ineffect the property : truncateText ',  */}
+          <EuiText size="s" className="eui-textTruncate" style={{ maxWidth: 150 }}>
+            {description}
+          </EuiText>
+        </EuiToolTip>
+      ),
+    },
+    {
+      field: 'lastUpdatedTime',
+      name: 'Last updated',
+      width: '25%',
+      truncateText: false,
+      render: (lastUpdatedTime: string) => {
+        return moment(lastUpdatedTime).format('MMM DD[,]YYYY [@] HH:mm:ss.SSS');
+      },
+    },
+
     {
       name: 'Actions',
       field: '',
       actions: [
+        {
+          name: 'Copy ID',
+          type: 'button',
+          description: 'Copy id',
+          'data-test-subj': 'workspace-list-copy-id-icon',
+          render: ({ id }: WorkspaceAttribute) => {
+            return (
+              <EuiButtonEmpty
+                onClick={() => handleCopyId(id)}
+                size="xs"
+                iconType="copy"
+                color="text"
+              >
+                <EuiText size="m">Copy</EuiText>
+              </EuiButtonEmpty>
+            );
+          },
+        },
         {
           name: 'Edit',
-          icon: 'pencil',
           type: 'icon',
+          icon: 'edit',
+          color: 'danger',
           description: 'Edit workspace',
-          onClick: ({ id }: WorkspaceAttribute) => handleSwitchWorkspace(id),
           'data-test-subj': 'workspace-list-edit-icon',
+          onClick: ({ id }: WorkspaceAttribute) => handleSwitchWorkspace(id),
+          render: ({ id }: WorkspaceAttribute) => {
+            return (
+              <EuiButtonEmpty
+                onClick={() => handleSwitchWorkspace(id)}
+                iconType="pencil"
+                size="xs"
+                color="text"
+              >
+                <EuiText size="m">Edit</EuiText>
+              </EuiButtonEmpty>
+            );
+          },
         },
         {
           name: 'Delete',
-          icon: 'trash',
-          type: 'icon',
+          type: 'button',
           description: 'Delete workspace',
-          onClick: (item: WorkspaceAttribute) => setDeletedWorkspace(item),
           'data-test-subj': 'workspace-list-delete-icon',
+          render: (item: WorkspaceAttribute) => {
+            return (
+              <EuiButtonEmpty
+                onClick={() => {
+                  setDeletedWorkspace(item);
+                }}
+                size="xs"
+                iconType="trash"
+                color="danger"
+              >
+                <EuiText size="m">Delete</EuiText>
+              </EuiButtonEmpty>
+            );
+          },
         },
       ],
     },
   ];
 
-  const workspaceCreateUrl = useMemo(() => {
-    if (!application || !http) {
-      return '';
-    }
-
-    const appUrl = application.getUrlForApp(WORKSPACE_CREATE_APP_ID, {
-      absolute: false,
+  useMemo(() => {
```

Comment:
actually you don't need a state for `featureFilters`, it can be derived from the workspace list, assume `workspace.useCase` is a string as I suggested
```typescript
  const featureFilters = useMemo(() => {
    const allUseCases = new Set<string>();
    newWorkspaceList.forEach((workspace) => {
      if (workspace.useCase) {
        allUseCases.add(workspace.useCase);
      }
    });
    return [...allUseCases];
  }, [newWorkspaceList]);
```